src/components/Navbar.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import CategoryFilter from './CategoryFilter';
+import '../styles/Navbar.css';
+
+const Navbar = () => (
+  <nav className="nav-bar" id="navbar">
+    <ul className="nav-list">
+      <li className="nav-list-item">
+        <h1 className="logo-h1">
+          <a className="logo-a" href="#navbar">Bookstore CMS</a>
+        </h1>
+      </li>
+      <li className="nav-list-item">
+        <a className="nav-a-selected" href="#books-table">BOOKS</a>
+      </li>
+      <li className="nav-list-item">
+        <CategoryFilter />
+      </li>
+    </ul>
+    <div className="user-div">
+      <i className="fas fa-user fa-2x" />
+
+    </div>
+  </nav>
+);
+
```

Comment:
- [ ] The font and spacing do not correspond to the design, kindly inspect the design and apply the necessary styling. The font "Montserrat" is google and free to use.
- [ ] Your filter component should have a title 'Category' as shown in the design